### PR TITLE
Updated compatibility matrix of 3.13

### PR DESCRIPTION
**Pull request by jchermoso**
Hello team,

This closes https://github.com/wazuh/wazuh-documentation/issues/2801

## Description
Updated compatibility matrix including Raspbian OS to indicate that it supports manager and agent of ARM architectures.

## Checks
- [x] It compiles without warnings.
- [x] Spelling and grammar. 
- [x] Used impersonal speech. 
- [x] Used uppercase only on nouns. 
- [x] Updated the `redirect.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).

Best regards,

Juan Carlos

### CODE REVIEW

File: source/installation-guide/compatibility_matrix/index.rst [hunk 1]
```diff
@@ -21,32 +21,33 @@ Wazuh manager
...
++-----------------------------------+-------------------------------+------------------------+--------+--------+
+|   RHEL 6 or newer                 | ✓                             | ✓                      |   ✓    |   ✓    |
++-----------------------------------+-------------------------------+------------------------+--------+--------+
+|   SUSE 12* or newer               | ✓                             | ✓                      |   ✗    |   ✗    |
++-----------------------------------+-------------------------------+------------------------+--------+--------+
+|   Ubuntu Precise/12.04 - 16.10    | ✓                             | ✓                      |   ✓    |   ✓    |
++-----------------------------------+-------------------------------+------------------------+--------+--------+
+|   Ubuntu Zesty/17.04 or newer     | ✓                             | ✓                      |   ✗    |   ✗    |
++-----------------------------------+-------------------------------+------------------------+--------+--------+
+|   Raspbian OS or newer            | ✓                             | ✗                      |   ✗    |   ✗    |
```
Comment (alberpilot) [inline source/installation-guide/compatibility_matrix/index.rst:49]: The list is alphabetically sorted. 
Raspbian OS has not a version, so newer has no sense without a Raspbian version.

File: source/_static/js/custom-redoc.min.js [hunk 2]
```diff
@@ -0,0 +1 @@
...ssList.remove('collapsed'); } else { this.classList.add('collapsed'); } }); checkMenu(btn); document.addEventListener('scroll', function() { const docHeight = document.body.clientHeight; const docWidth = document.body.clientWidth; const botomPos = window.innerHeight+window.scrollY; const menu = document.querySelector('.menu-content'); if ( docHeight - 30 <= botomPos && docWidth > 800 ) { menu.classList.add('reached-bottom'); } else { menu.classList.remove('reached-bottom'); } }); } } }; redocObserver = new MutationObserver(redocLoaded); redocObserver.observe(document.querySelector('redoc[spec-url]'), config); } function checkMenu(btn) { if ( btn ) { let menuButtonObserver = null; const config = {attributes: true}; const buttonStatus = function(mutationsList, observer) { for (i = 0; i < mutationsList.length; i++) { if ( mutationsList[i].attributeName == 'open' ) { if ( document.querySelector('redoc[spec-url] .menu-content').hasAttribute('open') ) { btn.classList.remove('collapsed'); } else { btn.classList.add('collapsed'); } } } }; menuButtonObserver = new MutationObserver(buttonStatus); menuButtonObserver.observe(document.querySelector('redoc[spec-url] .menu-content'), config); } }
```
Comment (alberpilot) [inline source/_static/js/custom-redoc.min.js:1]: Why this file?

File: source/_static/css/custom-redoc.min.css [hunk 3]
```diff
@@ -0,0 +1 @@
...idth: 801px){ #footer-redoc #footer-content { width: calc(100% - 260px); margin-left: auto; padding: 30px 45px; } .redoc-wrap .menu-content.reached-bottom { height: calc(100vh - 52px)!important; } .redoc-wrap .menu-content label.active { border-right: 1px solid #eaeaea; } body:not(.loading)::before { content: " "; background-color: #eaeaea; position: fixed; left: 0; top: 0; bottom: 0; width: 260px; } .redoc-wrap .api-content { padding-bottom: 79px; } footer#footer-redoc { position: relative; height: 0; bottom: 79px; right: 0; left: 0; } } @media (min-width: 1201px){ .redoc-wrap { margin-top: 0; } .redoc-wrap .menu-content, .redoc-wrap .menu-content.reached-bottom { top: 100px!important; height: calc(100vh - 100px)!important; } #header-redoc { width: 260px; height:100px; } #header-redoc .wazuh-logo-link img { max-width: 110px; } #header-redoc #links-to-home { width: 100%; justify-content: space-between; } #header-redoc #links-to-home .wazuh-logo-link { margin-right: 10px; } #header-redoc #links-to-home a.docs-link { padding-top: 6px; } #header-redoc .wazuh-logo-link img { max-width: 172px; } redoc[spec-url] .ddXLZs:not(:last-of-type)::after { left: 40px; width: calc(60% - 80px); } }
```
Comment (alberpilot) [inline source/_static/css/custom-redoc.min.css:1]: Why this file?

**Review by alberpilot — APPROVED** [review]
Please review the requested changes.

File: source/installation-guide/compatibility_matrix/index.rst (hunk 1 above)
Comment (jchermoso) [reply source/installation-guide/compatibility_matrix/index.rst:49]: Didn't know it, I'll fix both mistakes.

File: source/installation-guide/compatibility_matrix/index.rst [hunk 4]
```diff
@@ -21,32 +21,33 @@ Wazuh manager
...
++-----------------------------------+-------------------------------+------------------------+--------+--------+
+|   Debian Wheezy/7 or newer        | ✓                             | ✓                      |   ✓    |   ✓    |
++-----------------------------------+-------------------------------+------------------------+--------+--------+
+|   Fedora (>= 22)                  | ✓                             | ✓                      |   ✓    |   ✓    |
++-----------------------------------+-------------------------------+------------------------+--------+--------+
+|   OpenSUSE 42 or newer            | ✓                             | ✓                      |   ✗    |   ✗    |
++-----------------------------------+-------------------------------+------------------------+--------+--------+
+|   Oracle Linux 6 or newer         | ✓                             | ✓                      |   ✗    |   ✗    |
++-----------------------------------+-------------------------------+------------------------+--------+--------+
+|   Raspbian OS                     | ✓                             | ✗                      |   ✗    |   ✗    |
```
Comment (alberpilot) [inline source/installation-guide/compatibility_matrix/index.rst:41]: Raspberry Pi OS Stretch/9 or newer

**Review by alberpilot — CHANGES_REQUESTED** [review]
Please review the requested changes

File: source/installation-guide/compatibility_matrix/index.rst [hunk 5]
```diff
@@ -21,32 +21,33 @@ Wazuh manager
...
++-----------------------------------+-------------------------------+------------------------+--------+--------+
+|   Debian Wheezy/7 or newer        | ✓                             | ✓                      |   ✓    |   ✓    |
++-----------------------------------+-------------------------------+------------------------+--------+--------+
+|   Fedora (>= 22)                  | ✓                             | ✓                      |   ✓    |   ✓    |
++-----------------------------------+-------------------------------+------------------------+--------+--------+
+|   OpenSUSE 42 or newer            | ✓                             | ✓                      |   ✗    |   ✗    |
++-----------------------------------+-------------------------------+------------------------+--------+--------+
+|   Oracle Linux 6 or newer         | ✓                             | ✓                      |   ✗    |   ✗    |
++-----------------------------------+-------------------------------+------------------------+--------+--------+
+| Raspbian Pi OS Stretch/9 or newer | ✓                             | ✗                      |   ✗    |   ✗    |
```
Comment (alberpilot) [inline source/installation-guide/compatibility_matrix/index.rst:41]: This OS is not indented. you should indent Raspbian, starting at the same point that all others, and expand the entire table.

**Review by alberpilot — CHANGES_REQUESTED** [review]
Please review the requested changes

**Review by alberpilot — APPROVED** [review]
LGTM

**Pull request merged**